devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -201,12 +203,6 @@ private void createMavenWrapper(File createdPomFile) {
         }
     }
 
-    private SourceType determineSourceType(Set<String> extensions) {
```

Comment:
I initially debated whether moving this was a good idea, but I think it makes sense, :+1:

Reply:
it's just because I want to access it from external . for the web site. So it must not be in the mojo but in the command layer